TestHelmDeployPipelineRun is failing "a lot"

It is either flaky or something change in the image we use that makes the test fail on all PRs

```
I0604 07:01:08.915] --- FAIL: TestHelmDeployPipelineRun (95.57s)
I0604 07:01:08.916]     init_test.go:101: Create namespace arendelle-lpdmr to deploy to
I0604 07:01:08.916]     init_test.go:113: Verify SA "default" is created in namespace "arendelle-lpdmr"
I0604 07:01:08.916]     helm_task_test.go:230: Creating tiller service account
I0604 07:01:08.916]     helm_task_test.go:286: Creating Cluster Role binding tiller-6hb2d for helm
I0604 07:01:08.917]     helm_task_test.go:286: Creating Cluster Role binding default-tiller-pj2hh for helm
I0604 07:01:08.917]     helm_task_test.go:286: Creating Cluster Role binding default-tiller-c6wwv for helm
I0604 07:01:08.917]     helm_task_test.go:62: Creating Git PipelineResource go-helloworld-git
I0604 07:01:08.917]     helm_task_test.go:67: Creating Image PipelineResource go-helloworld-image
I0604 07:01:08.917]     helm_task_test.go:72: Creating Task create-image-task
I0604 07:01:08.917]     helm_task_test.go:77: Creating Task helm-deploy-task
I0604 07:01:08.917]     helm_task_test.go:82: Creating Pipeline helm-deploy-pipeline
I0604 07:01:08.917]     helm_task_test.go:87: Creating PipelineRun helm-deploy-pipeline-run
I0604 07:01:08.918]     helm_task_test.go:94: Error waiting for PipelineRun helm-deploy-pipeline-run to finish: pipeline run helm-deploy-pipeline-run failed!
I0604 07:01:08.918]     artifact_bucket_test.go:53: build logs {"level":"warn","ts":1559631657.6882505,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/test\" is not a valid GitHub commit ID"}
I0604 07:01:08.918]         {"level":"info","ts":1559631657.6889648,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}
I0604 07:01:08.918]     artifact_bucket_test.go:53: build logs {"level":"warn","ts":1559631591.789462,"logger":"fallback-logger","caller":"logging/config.go:65","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/test\" is not a valid GitHub commit ID"}
I0604 07:01:08.918]         {"level":"info","ts":1559631591.7898474,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}
I0604 07:01:08.918]     helm_task_test.go:104: PipelineRun execution failed; helm may or may not have been installed :(
```

/area testing

--- a/test/helm_task_test.go
+++ b/test/helm_task_test.go
@@ -180,8 +180,8 @@ func getHelmDeployTask(namespace string) *v1alpha1.Task {
 			tb.InputsParam("pathToHelmCharts", tb.ParamDescription("Path to the helm charts")),
 			tb.InputsParam("chartname", tb.ParamDefault("")),
 		),
-		tb.Step("helm-init", "alpine/helm", tb.Args("init", "--wait")),
-		tb.Step("helm-deploy", "alpine/helm", tb.Args(
+		tb.Step("helm-init", "alpine/helm:2.14.0", tb.Args("init", "--wait")),
+		tb.Step("helm-deploy", "alpine/helm:2.14.0", tb.Args(
 			"install",
 			"--debug",
 			"--name=${inputs.params.chartname}",
@@ -312,7 +312,7 @@ func helmCleanup(c *clients, t *testing.T, namespace string) {
 func removeAllHelmReleases(c *clients, t *testing.T, namespace string) {
 	helmRemoveAllTaskName := "helm-remove-all-task"
 	helmRemoveAllTask := tb.Task(helmRemoveAllTaskName, namespace, tb.TaskSpec(
-		tb.Step("helm-remove-all", "alpine/helm", tb.Command("/bin/sh"),
+		tb.Step("helm-remove-all", "alpine/helm:2.14.0", tb.Command("/bin/sh"),
 			tb.Args("-c", "helm ls --short --all | xargs -n1 helm del --purge"),
 		),
 	))
@@ -341,7 +341,7 @@ func removeAllHelmReleases(c *clients, t *testing.T, namespace string) {
 func removeHelmFromCluster(c *clients, t *testing.T, namespace string) {
 	helmResetTaskName := "helm-reset-task"
 	helmResetTask := tb.Task(helmResetTaskName, namespace, tb.TaskSpec(
-		tb.Step("helm-reset", "alpine/helm", tb.Args("reset", "--force")),
+		tb.Step("helm-reset", "alpine/helm:2.14.0", tb.Args("reset", "--force")),
 	))
 
 	helmResetTaskRunName := "helm-reset-taskrun"

